### ISSUE TITLE
Parallel Processing on GEOquery

### ISSUE THREAD

**Issue by ce-r** [seandavi/GEOquery#104]
hi sean,

i can't tell how much i love geoquery but it takes quite a long time to load data and i was wondering if you might tell me something. has anyone succeeded at parallel processing on it? 
i tried to use foreach and it only gave me a subset of what i requested. more specifically, of the 406 gsm ids, i only received data on 66 ids. 

best,
carlos rivas

**Comment by seandavi** [comment]
Hi, @ce-r. You should be able to use GEOquery in foreach loops. Note most time is spent in downloading, so there is a diminishing return to running in parallel. The 1 core vs 2 core difference you are seeing likely has to do with caching being applied in the second run. Finally, as for getting only a subset of the results, I cannot really comment without seeing code. 

I'm going to close this for how, but feel free to open a new issue or, better yet, ask on the bioconductor support site.
